requirements-base.txt

```diff
@@ -20,6 +20,8 @@ google-cloud-core==1.5.0
 googleapis-common-protos==1.52.0
 google-cloud-pubsub==2.2.0
 google-cloud-storage==1.35.0
+# Only necessary to prevent installing the latest version
+google-crc32c==1.1.2; python_version == '3.8'
```

Comment:
See https://github.com/googleapis/python-crc32c/issues/83

These google dependencies might end up moving into the dev requirements (#28305).